projects/plugins/jetpack/modules/masterbar/admin-menu/class-admin-menu.php

```diff
@@ -96,6 +103,10 @@ public function should_disable_links_manager() {
 	 * Adds My Home menu.
 	 */
 	public function add_my_home_menu() {
+		if ( self::CLASSIC_VIEW === $this->get_preferred_view( 'index.php' ) ) {
```

Comment:
We will need to handle in a separate Calypso PR a redirection to WP Admin when users use the site switcher, so that they don't land in Calypso My Home. Are we completely sure that adding the switcher in My Home is a good move?

![](https://cln.sh/C6Q8Mp+)

Reply:
> We will need to handle in a separate Calypso PR a redirection to WP Admin when users use the site switcher, so that they don't land in Calypso My Home

Oh good catch!

> Are we completely sure that adding the switcher in My Home is a good move?

This is what it was agreed on pd2C0z-3q-p2#comment-115

Reply:
Thanks for the references, I circled back here pd2C0z-3q-p2#comment-127

Reply:
We're finally excluding the quick switcher from My Home: https://simpleatomicp2.wordpress.com/2021/06/16/pod-call-hangout-jun-15th/#comment-146